LogcatCoreLib/src/main/java/info/hannes/logcat/LogcatActivity.kt

```diff
@@ -30,10 +30,8 @@ class LogcatActivity : AppCompatActivity() {
             // firstFragment.arguments = intent.extras
 
             // Add the fragment to the 'fragment_container' FrameLayout
-            supportFragmentManager.beginTransaction()
-                    .add(R.id.fragment_container, firstFragment).commit()
```

Comment:
In general, to avoid too much noise, I recommend to separate such changes in an own commit, the reduce following commits to real changes

Reply:
I don't understand fully, I grouped all the format changes in one commit, should I have made several commits, one for each file?